### ISSUE TITLE
CLML failed to compile on SBCL 1.4.14 on Windows 7 x64

### ISSUE THREAD

**Issue by navoj** [mmaul/clml#39]
It failed at clml.time-series. 

(UIOP/LISP-BUILD:CHECK-LISP-COMPILE-RESULTS NIL T T "~/asdf-action::format-actio
n/" ((#<ASDF/LISP-ACTION:COMPILE-OP > . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "clml.
time-series" "ts-util">)))
   error finding frame source: Bogus form-number: the source file has probably
                               changed too much to cope with.
   source: NIL
0]

**Comment by mmaul** [comment]
Hmm I never compiled on win/SBCL there might be a few dependencies that
might be problematic will take a look

**Comment by mmaul** [comment]
So good news and bad news...
Good news I was able to build on Windows 2016 datacenter edition. In fact quite I was amazed that everything just worked as Windows was not a platform I put much effort into supporting. The only issue I experienced was getting cl-ffi to load the "libcrypto-1_1-x64.dll" which is necessary for clssl which is necessary for the tests which use clml.utility.data:fetch to down load the test datasets from the github.io. All that aside clml did successfully compile on the windows platform, well on a windows platform.

Bad news it compiled on a Windows platform and I was not able to reproduce your problem. Of course I was not using Windows 7 and our setups may differ a bit.

Windows as a platform is probably not going to be the greatest due to SBCL multithreading support being fragile. I was going to point you at CCL as a compiler as it might have better Windows support however I tried that and that is having a problem loading a dependency 'array-operations'.

I might suggest you delete your C:\Users\%USERNAME%\AppData\Local\cache\common-lisp\sbcl-1.4.14-win-x64\C\Users\%USERNAME%\quicklisp\local-projects\clml and trying a recompile just to clear out the fasl cache.

I'll see if I can find a Win 7 to try to compile on to see if I can reproduce the issue.
